src/core/values/routes.ts

```diff
@@ -4,4 +4,14 @@ export enum Routes {
   SignUp = '/sign-up',
   ResetPassword = '/reset-password',
   Confirmation = '/confirm',
-}
\ No newline at end of file
+  Catalog = '/catalog',
+  Profile = '/profile',
+}
+
+export const RoutesList = [
```

Comment:
Cannot we get rid of the need of manually adding new routes to the `RouteList` by using `Object.values`:
```typescript
export const RoutesList = Object.values(route)
```

Reply:
@kirillkurko If memory does not change, then enum had problems with this when both keys and values were returned to Object.values. Perhaps this has already been corrected in new versions, you need to check

Reply:
Yes, it already works correctly

Reply:
Yep, it works fine now (actually I double-checked it before leaving my first comment)